docs/installation_guide.md

```diff
@@ -816,160 +877,171 @@ DEFAULT_SILBOPS_BROKER = 'http://pubsub.server.com:8080/silbops/CometAPI'
-   is also used for building the navigation bar.
-6. Run `python manage.py migrate; python manage.py collectstatic --noinput`
+-   Edit `urls.py`:
 
+    -   Replace the login endpoint:
+        -   Add the following import line at the beginning of the file:
+            `from wirecloud.fiware import views as wc_fiware`
+        -   Remove: `url(r'^login/?$', django_auth.login, name="login"),`
+        -   Add: `url(r'^login/?$', wc_fiware.login, name="login"),`
+    -   Add `social-auth-app-django` url endpoints at the end of the pattern list:
+        `url('', include('social_django.urls', namespace='social')),`
 
-[KeyRock's User and Programmers Guide]: https://fi-ware-idm.readthedocs.org/en/latest/user_guide/#registering-an-application
-[`FIWARE_PORTALS` setting]: #fiware_portals
+-   [Optional]: Change the `THEME_ACTIVE` setting to `wirecloud.fiwarelabtheme`. This theme is the one used by the
+    FIWARE Lab's Mashup portal.
+-   [Optional]: Provide a [`FIWARE_PORTALS` setting](#fiware_portals). This setting is used for signing out from other
+    portals at the same time the user sign out from WireCloud, providing a single sign out experience. This setting is
+    also used for building the navigation bar.
+-   Run `python manage.py migrate; python manage.py collectstatic --noinput`
 
+[keyrock's user and programmers guide]:
+    https://fi-ware-idm.readthedocs.org/en/latest/user_guide/#registering-an-application
+[`fiware_portals` setting]: #fiware_portals
 
 ### Enabling the real-time synchronization support
 
-WireCloud 1.0 adds experimental support for real time synchronization through
-web sockets.
+WireCloud 1.0 adds experimental support for real time synchronization through web sockets.
 
 The steps for enabling this support are the following:
 
-1. Install [Django channels](https://channels.readthedocs.io/en/latest/):
-  ```
-  $ pip install channels
-  ```
-2. Add `channels` and `wirecloud.live` into the `INSTALLED_APPS` setting in the
-    `settings.py` file.
-3. Configure the channels framework by configuring the `CHANNEL_LAYERS` setting.
-    For example, you can make use of the following configuration:
-
-    ```python
-    CHANNEL_LAYERS = {
-        "default": {
-            "BACKEND": "asgiref.inmemory.ChannelLayer",
-            "ROUTING": "wirecloud.live.routing.channel_routing",
-        },
-    }
-    ```
-
-Once done those steps, you will be able to use WireCloud using the runserver
-command. Take into account that is not possible to deploy WireCloud when using
-the real-time synchronization support using the standar Apache configuration.
-We are working on providing better documentation and examples on how to deploy
-WireCloud in this case for a production ready environment, in the meantime, you
-can take a look into the [Django channels
-documentation](https://channels.readthedocs.io/en/latest/deploying.html).
+-   Install [Django channels](https://channels.readthedocs.io/en/latest/):
+
+```bash
```

Comment:
Not done, since `<zbr>` should be sufficient:

![Screenshot 2019-03-26 at 15 06 06](https://user-images.githubusercontent.com/3439249/55008326-c51c3080-4fd8-11e9-823e-830b0457bfeb.png)